### ISSUE TITLE
leaflet plugins for layer control

### ISSUE THREAD

**Issue by timelyportfolio** [tim-salabim/rspatial#10]
I don't think any of these on their own satisfy layer control from [Visualize.Rmd](https://github.com/tim-salabim/rspatial/blob/master/chapters/visualise/mapview.Rmd) (https://github.com/tim-salabim/rspatial/commit/b6726f708255cda2663ee27adde3ba973f53d4a7), but I thought we should have a list of partial solutions.

- [Leaflet.AutoLayers](https://github.com/aebadirad/Leaflet.AutoLayers)
- [Leaflet.layerscontrol-minimap](http://jieter.github.io/Leaflet.layerscontrol-minimap/)
- [Leaflet.Sidebar v2](https://turbo87.github.io/sidebar-v2/examples/)

Should I place this in the Wiki section instead, so we can inline edit?

**Comment by tim-salabim** [comment]
Yes, I have seen those, but none really satisfy the GIS-like control mechanisms we should provide for a good user experience. From what I've seen so far, all the features I included in the solution proposed in [Visualize.Rmd](https://github.com/tim-salabim/rspatial/blob/master/chapters/visualise/mapview.Rmd) are available so it's a matter of bundling existing features into one leaflet plugin.
